Misc/NEWS.d/next/Documentation/2017-10-08-23-02-14.bpo-31537.SiFNM8.rst

```diff
@@ -0,0 +1,2 @@
+Fix incorrect usage of `get_history_length` in readline documentation
```

Comment:
Please use double backticks: \`\`get_history_length\`\`

Single backtick means Sphinx will use the value of the `default_role` setting to decide what to do with a markup like \`foo bar\` and it can cause regressions in documentation if someone changes the value of `default_role` by using the `default-role` directive:

```rst
`foo bar`

.. default-role:: subscript

`spam eggs`
```

will be rendered as

```html
<p><cite>foo bar</cite></p>
<p><sub>spam eggs</sub></p>
```

See http://docutils.sourceforge.net/docs/ref/rst/directives.html#default-role for details.